packages/components/file-card/src/utils/getFileInfo.js

```diff
@@ -0,0 +1,7 @@
+export function getFileInfo(fileUrl) {
+  const url = document.createElement('a');
+  url.href = fileUrl;
+  const urlParams = url.search.substring(1);  
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the useless trailing whitespaces at the end of this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3ATrailingWhitespace)